qiita_pet/templates/sitebase.html

```diff
@@ -145,8 +146,7 @@
                 <li><a href="/study/public/">View Public Studies</a></li>
               </ul>
             </li>
-            {% set user_tmp = handler.user() %}
-            {% set user_level = user_tmp.level if user_tmp is not None else None %}
+            {% set user_level = user.level %}
```

Comment:
This is needed to show the admin or dev menu at the top so it's needed in every page. Easier to put it in the template because of that.